conference.js

```diff
@@ -1106,11 +1113,76 @@ export default {
     videoSwitchInProgress: false,
 
     /**
+     * This fields stores a handler which will create a Promise which turns off
+     * the screen sharing and restores the previous video state (was there
+     * any video, before switching to screen sharing ? was it muted ?).
+     *
+     * Once called this fields is cleared to <tt>null</tt>.
+     * {@type Promise|null}
```

Comment:
yes, it should be a function, not sure how to write that...
`{@type function()|null}`
will that work ?